atlp-rwanda/atlp-pulse-fn
# fix CTA button get started

# PR Description

1. This PR is for fixing CTA button(Get Started)
2. It will redirect a user to another page(Create new Organization)
3. Fix also some issues on sidebar for both users.

# How has this been tested?

1. Open The review app link.
4.  Go to homepage and check if `Get Started` redirect you to other Page(Create a new organization).
5. Check if it works in all kind of devices(Responsiveness).


# Screenshots (If appropriate)

<img width="1423" alt="Screen Shot 2022-11-08 at 19 20 58" src="https://user-images.githubusercontent.com/100874204/200632807-9c78f877-49f0-458a-b94b-630862e00c70.png">
